index.html

```diff
@@ -119,6 +129,22 @@
 						[coords[2], coords[3]]
 					];
 					map.fitBounds(bbox, options={padding: 10, duration: 5000});
+					// find the middle of the bbox to put a popup there
+					var middle = new mapboxgl.LngLat(
+						(parseFloat(coords[0]) + parseFloat(coords[2])) / 2,
+						(parseFloat(coords[1]) + parseFloat(coords[3])) / 2
+					);
+					var data = {
+						SchDistNam: coords[4],
+						EstADAIn: coords[5]
+					};
+					// close any existing popup[s]
+					document.querySelectorAll('.mapboxgl-popup').forEach(e => e.remove());
+					// draw the new one
+					new mapboxgl.Popup()
+						.setLngLat(middle)
+						.setHTML(fillpopup(data))
+						.addTo(map);
```

Comment:
This is what actually draws a popup on zoom.  Because there's no click event to get coordinates from, I locate it at the middle of the polygon's bounding box.

Reply:
...hence the issue with the crazy gerrymandered ISDs. Makes sense.